src/store/flow/resource.ts

```diff
@@ -163,6 +163,15 @@ export function findResourceVariantOverModesOn(
   filter: IResourceDefinitionVariantOverModesFilter,
 ): IResourceValue {
   // b/c we do explicit partial matching on modes
+  // why do we do partial matching on modes?
+
+  // todo: is there a way that we could do an inclusive+additive search here?
```

Comment:
Is the TODO still relevant or just a perso note during implementation?
Might be useful to track it somewhere in JIRA